src/app/business/block/file-share-detail/file-share-detail.component.ts

```diff
@@ -238,11 +242,15 @@ export class FileShareDetailComponent implements OnInit{
                         description: item.description? item.description: "--",
                         level: item.accessCapability,
                         createdAt: Utils.formatDate(item.createdAt),
-                        updatedAt: Utils.formatDate(item.updatedAt)
+                        updatedAt: Utils.formatDate(item.updatedAt),
+                        date: new Date(Utils.formatDate(item.updatedAt)).getTime()
                     }
                     this.acls.push(acl);
                 }
             })
+            this.acls.sort((a,b)=>{
```

Comment:
please use variable names that are readable. Using `a` and `b` is not very helpful.

Reply:
I have just modified it. Please check it.